pkg/metrics/features/policy.go

```diff
@@ -180,63 +147,27 @@ func ruleTypePortRules(rf *RuleFeatures, portRules api.PortRules) {
 	}
 }
 
-func ruleType(r api.Rule) RuleFeatures {
+func ruleType(r types.PolicyEntry) RuleFeatures {
 
 	var rf RuleFeatures
 
-	rf.NonDefaultDeny =
-		r.EnableDefaultDeny.Ingress != nil && *r.EnableDefaultDeny.Ingress ||
-			r.EnableDefaultDeny.Egress != nil && *r.EnableDefaultDeny.Egress
-
-	for _, i := range r.Ingress {
-		ruleTypeIngressCommon(&rf, i.IngressCommonRule)
-		if !rf.allFeaturesPortRules() {
-			ruleTypePortRules(&rf, i.ToPorts)
+	rf.Deny = r.Deny
+	rf.MutualAuth = r.Authentication != nil
+	rf.NonDefaultDeny = r.DefaultDeny
+	rf.ToFQDNs = len(types.FromPeerSelectorSlice[api.FQDNSelector](r.L3)) > 0
+	rf.L3 = len(r.L3) > 0 && !rf.ToFQDNs
```

Comment:
This doesn't seem right, but the bug seems to be present in the tree already. FQDNSelectors are an L3 selector so it's weird that feature reporting doesn't treat it as such.
```suggestion
	rf.L3 = len(r.L3) > 0
```

Reply:
(Maybe worth deferring this fix to a future PR so we can pull in the relevant authors to understand whether this bugfix needs backporting)

Reply:
I'll mark this resolved as this is existing behavior. Let's revisit in a future PR.